eventservice/api/utils/group_auth.py

```diff
@@ -4,6 +4,8 @@
 from flask import g
 
 logger = logging.getLogger(__name__)
+ALL_GROUP_EVENTS = 'all_group-events'
+ALL_EVENTS = 'all-events'
```

Comment:
@ywkim312, Currently, there is only `all_group-events` permission as this feature is limited to group events. I think you could remove this second permission string and the related case below. What do you think?

Reply:
I made it for the future use but I can delete these

Reply:
removed, however, I had couple methods related to this one in events.py file. Should I remove those as well? Or maybe we can keep those methods and just make this line (variable) reactive when the variables actually comes alive so those method can be used?

Reply:
Hi, @ywkim312, After going through your comments, I think it's better to remove the use of `all-events` permission at all places. If such permission gets added in the future, we can bring these back. What do you think?

Reply:
pushed